systems/betSystem.js

```diff
@@ -105,19 +91,27 @@
         
         if (betStatus && !betClosed) {
             
-            var closedPot = 0;
+            var closedPot = betPot;
+            var langString = '';
+            langString += betPot + ' Quoten:';
+            var quotes = calcQuotes();
             betClosed = true;
-            
-            for (var i in betTable) {
-                var bet = betTable[i];
-                closedPot += bet.amount;
+
+            for (var i = 0; i < quotes.length; i++) {
+                var element = quotes[i];
+                langString += element.option.toUpperCase()  + ': ' + element.quote + ' (' + element.bets + ')';
+                if ((i-1) == quotes.length) {
+                    langString += '.';
+                } else {
+                    langString += ' | ';
+                }
             }
 
             $.log.file('betSystem', 'Bet closed by ' + event.getSender() + '.');
             $.say($.lang.get('betsystem.closed', closedPot));
         }
         else {
-                // TODO
+                $.say($.whisperPrefix(sender) + $.lang.get('betsystem.close'));
```

Comment:
Shouldn't ``betsystem.close`` be ``betsystem.closed.nobet`` ?
Looks greate otherwise.